README.md

```diff
@@ -83,6 +83,13 @@ NOTE: Currently this option is used for ALL partitions in upstream topic(s).
 
 In case there is a need to discard committed offsets, pick a new group ID.
 
+### Offset Commit Policy
+The `Offset_commit_policy` specify how upstream consumer manages the offset per partition per topic.
```

Comment:
per topic-partition

`topic-partition` is mostly adopted term to refer a `{Topic, Partitin}`